src/main/java/dre/Dre.java

```diff
@@ -0,0 +1,49 @@
+package dre;
+
+import dre.exception.DreException;
+import dre.ui.Ui;
+import dre.storage.Storage;
+import dre.parser.Parser;
+import dre.task.TaskList;
+import dre.command.Command;
+
+public class Dre {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
```

Comment:
Perhaps it would be better if you made these three variables final. (Remember to CAP all letters :) )